manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: JTL-SoftwareGmbH.JTL-Wawi
+PackageVersion: 1.7.8.1
+PackageLocale: en-US
+Publisher: JTL-Software GmbH

```

```suggestion
Publisher: JTL-Software-GmbH
```
Must match Control Panel / Apps&Features